src/main/java/at/hannibal2/skyhanni/SkyHanniMod.kt

```diff
@@ -69,9 +68,8 @@ object SkyHanniMod {
         screenToOpen?.let {
             screenTicks++
             if (screenTicks == 5) {
-                val title = Minecraft.getMinecraft().currentScreen?.getTitle()
                 MinecraftCompat.localPlayer.closeScreen()
-                OtherInventoryData.close(title)
+                OtherInventoryData.close()
```

Comment:
have you tested this? localPlayer.closeScreen changes the current screen, threfore `InventoryCompat.getOpenChestName()` cant show the current name again. the code you removed here was as a workaround for this problem

Reply:
ups not this line, one below